### ISSUE TITLE
pyrmont.launchpad's "Get GöETH" links to rate-capped faucet

### ISSUE THREAD

**Issue by chrishobcroft** [ethereum/eth2.0-deposit#417]
# Summary

When using prymont.launchpad, the "Connect Wallet" step provides a link to "Get GöETH". This links to the [Goerli: Authenticated Faucet](https://faucet.goerli.mudit.blog/), which can provide a maximum of 1 Ethers every 1 day. A user would need to wait 32 days to use this service to obtain the 32 Ethers in order to proceed to the next step.

# Proposed Solution

This is a hard one. Nobody probably wants to run a service to give out lots of GöETH.

It used to be possible to get it from Prysmatic Labs' Discord server's #request-goerli-eth channel... but this faucet ran out around 2021-05-26, and normal services have not yet resumed.

Aside from this, I have so far been unable to find anywhere to get 32 GöETH, in order to launch a Validator on a testnet.

# Screenshots

Showing the link to "Get GöETH":

![image](https://user-images.githubusercontent.com/2212651/126870335-d9b96baf-4974-47d6-9abd-70d927e50793.png)

Showing the rate-cap on getting GöETH through this suggested channel.

![image](https://user-images.githubusercontent.com/2212651/126870397-ff2e7786-3c81-42b6-b7a9-c9f0256d0db6.png)

**Comment by samajammin** [comment]
@CarlBeek @wackerow @hwwhww - do we know of any faucets that supply 32 GöETH ETH?

**Comment by chrishobcroft** [comment]
@samajammin it is possible to get a 32 GöETH deposit made on your behalf via the ethstaker discord server. The process involves sharing the full transaction data for making the deposit. Extracting this transaction data requires the HTML of a page on pyrmont.launchpad to be edited, in order to activate a button, to allow a user to proceed to the next step.

While this process does work, it could be much easier.

**Comment by hwwhww** [comment]
I have no idea. 😅 

@q9f Any chance you know any faucets that supply 32 GöETH ETH now?

**Comment by q9f** [comment]
> I have no idea. sweat_smile
> 
> @q9f Any chance you know any faucets that supply 32 GöETH ETH now?

No, I'm in the same boat. 😓  See also: https://github.com/goerli/testnet/issues/85 https://github.com/goerli/testnet/issues/84

The problem is the huge abuse of the faucets. The Eth2 deposit contracts for the testnets put _some_ pressure on Goerli but not too much, nothing Goerli wouldn't be able to handle. But apparently, the Swarm tests on Goerli killed it because people misunderstood (?) the procedure and where offering many people real money OTC to buy Goerli Eth, i.e., @Buttaa https://twitter.com/Butta_eth/status/1412428424816349185

The faucet comes with social authentication, captchas, IP-based limits, and ETH-address-based limits, still, there was a massive pressure on the faucets. @maxsam4 did a lot of work to keep it up but I guess now the only solution was to reduce the amounts that can be requested from the faucets further.

That said, the reserves of the Goerli whales are being drained as well. I personally handed out more than 30 million Goerli Eth in the last 12 months. At this rate, Goerli would probably run dry anytime soon.

The most straight-forward solution would be, in my opinion, the following (hear me out, don't judge):
1. Give Goerli a value and list it on Exchanges or bridge it to mainnet as ERC-677 token. There is no better way to guarantee liquidity long-term.
2. Use Goerli only for larger applications that require the massive existing infrastructure and can afford spending money on usage (looking at you, Swarm).
3. Launch a subsequent testnet that will allow for long-term stable and free testing (think: Goerli v2).

This comes with huge downsides though and I guess that's why nobody has done it so far:
1. Adding a price tag to something that has been promised for free, see also Mudit's comments here: https://twitter.com/Mudit__Gupta/status/1412529001860534274
2. The Goerli Eth distribution is largely arbitrary and giving it value would be extremely unfair.
3. There are many legal uncertainties.

Besides, an incentivized testnet would be actually useful for some cases, e.g., https://twitter.com/DeanEigenmann/status/1419289668324057095 - It just needs to be planned and rolled out very carefully.

But I don't want to derail the conversation. Maybe PoA-testnets have an implicit end-of-life date built in and we never thought that Goerli would be so broadly used back in the day when we launched it. Maybe a new testnet would indeed be an idea worth exploring? Downside here is that all infrastructure has to be duplicated. Or, we hardfork Goerli and increase total supply...

**Comment by maxsam4** [comment]
@q9f As a temporary fix, I can bump the faucet to drip a bit over 32 ETH. 

Apart from the rate limits you mentioned, I've added the following stuff recently
- Twitter account age, followers, and tweets requirement. This has basically stopped the current bots that were creating new accounts and instantly requesting funds.
- WebSocket request limits and timeout. Without this, the IP rate limits weren't as effective.

Someone with enough motivation can still spam the faucet but given that ethswarm is no longer actively giving GoETH value, I doubt anyone will go to the troubles.

I propose that I raise the drip limit to just over 32 ETH while we keep looking for an alternative solution. If we see the spam return, I'll lower the limits again.

**Comment by maxsam4** [comment]
FYI, The drip limit on https://faucet.goerli.mudit.blog/ has been increased. You can now get up to 37.5 ETH in a single request.

**Comment by maxsam4** [comment]
I've dropped the max drip to 6.25 ETH to extend the life of the faucet. It has already dropped 60% of its remaining balance and is being used more frequently than ever now.

@q9f we need to eventually think about minting more goerli eth that can be disbursed.

**Comment by q9f** [comment]
I still have 23kk. Eventually, if we run out, we might want to launch a clean, merged testnet anyways.

**Comment by cynicalyeezus** [comment]
Hello, I’m new to this site and just made an account. I’ve been interested in crypto and blockchain technology and more recently have been reading about upcoming “web3.0”. I have a few wallets and have been trying to learn about the different methods to get crypto on the different networks. I was able to get the 6.25 Eth on the Goerli test network from the faucet drip! I was wondering if you can run a node once I have 32 Eth? Thanks

**Comment by chrishobcroft** [comment]
Hi @cynicalyeezus - welcome! Yes, once you have 32 goETH (plus some goETH to pay the gas), you would be able to deposit into a contract on the Goerli testnet, which would allow you to run a Proof-of-Stake Validator node on a testnet.

Do you have any more questions?

**Comment by cynicalyeezus** [comment]
That's great to know! I waited 2 days to try the drip again but it still said wait 48 hours. Do you guys have any information or tips to get more Ethereum? And also how to start a node on my computer. Thanks

**Comment by JonathanSAW19** [comment]
@maxsam4 
good night sir, now how to get GOERLI TESTNET for free? 
various sites for my faucet, all can't. 
please answer. thank you

**Comment by q9f** [comment]
> @q9f we need to eventually think about minting more goerli eth that can be disbursed.

Please check out https://github.com/goerli/testnet/issues/97

**Comment by kingliu8888** [comment]
I support this effort. Thanks!

**Comment by Jiangzy520** [comment]
> > 我不知道。  汗水微笑 
> > @q9f 你有没有机会知道现在提供 32 GöETH ETH 的水龙头？
> 
> 不，我在同一条船上。 😓 参见： [goerli / testnet #85 ](https://github.com/goerli/testnet/issues/85) [goerli / testnet #84 ](https://github.com/goerli/testnet/issues/84)
> 
> 问题是水龙头的大量滥用。 测试网的 Eth2 存款合约给 Goerli 带来 _了一些 _压力，但压力不大，没有 Goerli 无法处理的。 但显然，对 Goerli 的 Swarm 测试杀死了它，因为人们误解了（？）程序以及向许多人提供真钱场外交易以购买 Goerli Eth 的地方，即 @Buttaa [https://twitter.com/Butta_eth/status/1412428424816349185 ](https://twitter.com/Butta_eth/status/1412428424816349185)
> 
> 水龙头带有社交身份验证、验证码、基于 IP 的限制和基于 ETH 地址的限制，但水龙头仍然承受着巨大的压力。 @maxsam4 做了很多工作来保持它，但我想现在唯一的解决方案是进一步减少可以从水龙头请求的数量。
> 
> 也就是说，Goerli 鲸的储备也正在枯竭。 在过去的 12 个月里，我个人分发了超过 3000 万个 Goerli Eth。 以这样的速度，歌尔力可能很快就会枯竭。
> 
> 在我看来，最直接的解决方案是以下（听我说，不要评判）：
> 
>     1. 给 Goerli 一个值并在交易所上架或将其作为 ERC-677 代币桥接到主网。  没有更好的方式来保证长期流动性。
> 
>     2. 仅将 Goerli 用于需要大量现有基础架构并且能够负担得起使用费用的大型应用程序（看着你，Swarm）。
> 
>     3. 启动后续测试网，允许长期稳定和免费的测试（想想：Goerli v2）。
> 
> 
> 不过，这有很大的缺点，我想这就是为什么到目前为止没有人这样做的原因：
> 
>     1. 为免费承诺的东西添加价格标签，另请参阅 Mudit 的评论： [https ](https://twitter.com/Mudit__Gupta/status/1412529001860534274)
> 
>     2. Goerli Eth 的分布在很大程度上是任意的，赋予它价值是非常不公平的。
> 
>     3. 有很多法律上的不确定性。
> 
> 
> 此外，激励测试网在某些情况下实际上是有用的，例如， [https ](https://twitter.com/DeanEigenmann/status/1419289668324057095)- 它只需要非常仔细地计划和推出。
> 
> 但我不想破坏谈话。 也许 PoA 测试网内置了一个隐含的生命周期结束日期，我们从未想过 Goerli 在我们推出它的那天会被如此广泛地使用。 也许一个新的测试网确实是一个值得探索的想法？ 这里的缺点是所有基础设施都必须复制。 或者，我们硬分叉 Goerli 并增加总供应量......
sir i buy goerli
